CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
You can remove this.

Reply:
I have removed the CODE_OF_CONDUCT.md file from this repo.

Please review